src/containers/BooksForm.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const BooksForm = () => {
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  return (
+    <div>
+      <input />
+      <select>
+        {categories.map((cat) => (
+          <option key={cat} value={cat}>{cat}</option>
+        ))}
+        ;
+      </select>
+      <button type="submit"> Submit</button>
+
+    </div>
```

Comment:
According to the instructions on pathwright and getting data from user implementation, you need to use the form tag.
![image](https://user-images.githubusercontent.com/31897434/124518528-893ebb00-ddde-11eb-9b86-cb7901eeeb62.png)